tests/testthat/test-animations.R

```diff
@@ -0,0 +1,23 @@
+context("Animations")
+
+foo <- data.frame(
+  series = rep(1:4, each = 10),
+  x = rep(seq.Date(from = as.Date("2001-01-01"),
+                      by = "quarter",
+                      length.out = 10)),
+  y = c(-0.4, 1.1, 1.4, -0.1, 0.4, 0.7, 1.5, -0.2, 0.1, -0.2,
+        -0.5, -0.9, -0.1, -0.2, -0.6, 1.3, 0.9, -0.5, 0.7, -1,
+        -1.5, -6, -8.6, -6.6, -5, -12, -18, 17, 13, 5,
+        -1.7, 1.5, -1.8, 2.4, -11.7, 10.1, 1.4, -4, -4.2, -4.8))
+
+test_that("Simple animations", {
+  foo_list <- split(foo, foo$series)
+
+  gg_list <- lapply(foo_list, function(data) {
+    gg <- arphitgg(data = data, aes = agg_aes(x = x, y = y, group = series)) +
+      agg_line() + agg_ylim(-20, 20, 5)
+  })
+
+  gif_file <- agg_slides(gg_list, filename = tempfile("ani", fileext = ".gif"))
```

Comment:
`agg_slides` doesn't return the filename anymore, so this test doesn't work. Could just assign the tempfile name to `gif_file` first and use that in both `check_gif` and `agg_slides(... filename = gif_file)`

Reply:
`agg_slides` still returns the filename if it is given one. I've left it that way because that's the return value of `imagemagick::image_write`.

So the test passes on my computer with that code. If it doesn't pass on your computer, happy to make the change.

Reply:
My mistake, you are absolutely right. I obviously mucked something stupid up when I ran this, sorry!